config/locales/en.yml

```diff
@@ -68,6 +68,8 @@ en:
             delivery_site:
               blank: Choose a delivery site
               inclusion: Choose a delivery site
+              nasal_spray_must_be_nose: Must be nose when delivery method is nasal spray
+              injection_cannot_be_nose: Cannot be nose when delivery method is intramuscular or subcutaneous
```

Comment:
Should I have a specific error message for each injection type? e.g "Cannot be nose when delivery method is intramuscular" and "Cannot be nose when delivery method is subcutaneous"